### PR TITLE
Add API documentation for project creation and run log retrieval

### PR DESCRIPTION
**Pull request by chrismyang**

### CODE REVIEW

File: api_blueprint.apib [hunk 1]
```diff
@@ -79,6 +79,27 @@ Your API Endpoint is a wrapper on top of a function in your Domino-hosted code.
...
+    + Attributes
+        + projectName: `winequality` (string, required) - The name of the project to create
+        + ownerOverrideUsername: `domino` (string, optional) - The owner to create the project for, typically for an
+          Organization you are a member of.  If not provided, the owner is the user making the creation request.
+        + useDefaultIgnoreFile: `true` (boolean, optional) -  If true, use the Domino application's default `.dominoignore`
+          file in the project. Otherwise, the project is not created with a `.dominoignore` file.
+
++ Response 200 (application/text)
+
+        The string "Created project {owner}/{projectName} successfully."
```
Comment (marks) [inline api_blueprint.apib:101]: @chrismyang I think this should not include `"The string"` part

**Review by marks — APPROVED** [review]
This looks good to me. Left one comment about the response formatting for the project creation API

**Comment by marks** [comment]
@chrismyang - will let you merge when the time is right.

One more feedback too: is the project creation API available since, say, 2.0 or is it new?

File: api_blueprint.apib (hunk 1 above)
Comment (chrismyang) [reply api_blueprint.apib:101]: Got it -- so weird we just return a random string :)

**Comment by chrismyang** [comment]
Nope it's old like...from 2014.

**Comment by marks** [comment]
@chrismyang - shall we merge this?

**Comment by marks** [comment]
@njablonski can you publicly document the stdout API above?

**Comment by marks** [comment]
Closing in favor of #19

**Pull request closed without merging**